### PR TITLE
Ticket two

### PR DESCRIPTION
**Pull request by AnvarjonShavqiyev**
- Created an endpoint to fetch user details by ID.
- Created a user profile component.
- Fetched and displayed user details when navigating to the profile page.

Home page:
image
User profile page:
image

You can go user profile page with click name of user on table, or click account which located navbar for this user.

### CODE REVIEW

File: my-react-app/src/pages/Profile/Profile.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import UserInformation from '../../components/userInformation/UserInformation'
+
+const Profile:React.FC = () => {
+  const {id} = useParams()
+  const dispatch = useDispatch<AppDispatch>() 
+  const thisUser = useSelector((state:RootState) => state.users.thisUser)
+  useEffect(()=> {
+    dispatch(getById({ id }))
+  },[])
+  console.log(thisUser)
```
Comment (VladKetris) [inline my-react-app/src/pages/Profile/Profile.tsx:18]: do you still need it here ?

File: my-react-app/src/pages/Profile/Profile.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import { getById } from '../../redux/features/UsersSlice'
+import './Profile.scss'
+import ProfileNav from '../../components/profileNav/ProfileNav'
+import UserInformation from '../../components/userInformation/UserInformation'
+
+const Profile:React.FC = () => {
+  const {id} = useParams()
+  const dispatch = useDispatch<AppDispatch>() 
+  const thisUser = useSelector((state:RootState) => state.users.thisUser)
+  useEffect(()=> {
```
Comment (VladKetris) [inline my-react-app/src/pages/Profile/Profile.tsx:15]: please add empty line before and after hooks

File: my-react-app/src/redux/features/UsersSlice.ts [hunk 3]
```diff
@@ -5,19 +5,25 @@ import instance from '../../service/api/axios';
...
 
 interface UpdateUserPayload {
   id: string;
   newData: Partial<User>;
 }
 
-interface DeleteUserPayload {
-  id: string;  
+interface UserPayload {
+  id: any;
```
Comment (VladKetris) [inline my-react-app/src/redux/features/UsersSlice.ts:22]: please avoid using any type

File: my-react-app/src/pages/Profile/Profile.tsx (hunk 2 above)
Comment (AnvarjonShavqiyev) [reply my-react-app/src/pages/Profile/Profile.tsx:15]: i got it

File: my-react-app/src/redux/features/UsersSlice.ts (hunk 3 above)
Comment (AnvarjonShavqiyev) [reply my-react-app/src/redux/features/UsersSlice.ts:22]: I understood

File: my-react-app/src/pages/Profile/Profile.tsx (hunk 1 above)
Comment (AnvarjonShavqiyev) [reply my-react-app/src/pages/Profile/Profile.tsx]: fixed

**Comment by VladKetris** [comment]
why don't you fixed any type and line breaks ?

**Review by VladKetris — APPROVED** [review]

**Pull request merged**